pkg/scheduler/scheduler.go

```diff
@@ -110,6 +111,39 @@ func (s *Scheduler) internalPreemption() {
 	}
 }
 
+func (s *Scheduler) internalInspectOutstandingRequests() {
+	for {
+		time.Sleep(1000 * time.Millisecond)
+		s.inspectOutstandingRequests()
+	}
+}
+
+func (s *Scheduler) inspectOutstandingRequests() {
```

Comment:
Suggest to rename it to `UpdateUnschedulableStateForOutstandingRequests`

Reply:
I found `UpdateUnschedulableStateForOutstandingRequests` is also a bit confusing. Instead of updating this function name, I added more code comment to explain what it does:

```
// inspect on the outstanding requests for each of the queues,
// update request state accordingly to shim if needed.
// this function filters out all outstanding requests that being
// skipped due to insufficient cluster resources and update the
// state through the ContainerSchedulingStateUpdaterPlugin in order
// to trigger the auto-scaling.
```